### ISSUE TITLE
local_settings not on docker host

### ISSUE THREAD

**Issue by rhoerbe** [Emergya/peer-dockerfiles#5]
gunicorn.py and localsettings.py should be taken from the docker host, instead of providing it with the image.

It should be documented that the deployer needs to provide these files before running a container.

**Comment by enriquepablo** [comment]
The  thing is that local_settings.py is also needed at the time of building the image, to set up the db. So we need to edit it before building the image, and add it to the image; and then we may need to again edit it before running the container, to change runtime settings, and that's the reason it is provided with the image.
At the moment, the prod-env/local_settings/ dir is created when creating a container from the image, so the deployer does not need to provide them.
A solution might be to expose the local_settings/ volume from the container to peer/config/ (from which they are added to the image) rather than to prod-env/, to not have duplicate config files. The problem I saw with this was that the settings in peer/config/ are under git control, but this problem may be less grave than the confusion from having duplicate config files ?

**Comment by rhoerbe** [comment]
Agreed. Would it be useful you separate out a local_db_settings.py that is used at image-build time, and leave the rest external?

**Comment by enriquepablo** [comment]
I'm not sure I see your goal, but I don't see that the workflow for the deployer would be any simpler

**Comment by rhoerbe** [comment]
Maybe I am missing something. What I want is that config information should be dynamic, without image rebuilding, and persistent across updates, i.e. a git pull should not overwrite my settings. So if I change my LOGGING settings and restart gunicorn, it should work.

**Comment by enriquepablo** [comment]
That's how it works now, and not something we should loose with the deduplication of settings files.

Git does not overwrite local changes, but it may be a nuisance, having to git stash your settings to update the repo, and having to be careful to not push your settings to github.

How about this:
 * Before building the image, the user can edit `peer/config/local_db_settings.py`.
 * Before building the image, there are `peer/config/local_settings.pre-image` and `peer/config/gunicorn.pre-image`, that are added to the image.
 * When the container is run for the first time, the "pre-image" files are mved to `local_settings/<name>.py` and exposed at `prod-env/local_settings/`, and the user is then allowed to edit them and restart gunicorn/the container.

I think that this way we separate db config from runtime config, we don't mess with git, and we don't have duplicate filenames. What do you think?

**Comment by rhoerbe** [comment]
Agreed, this should do it. 

Von meinem iPhone gesendet
